doc/man3/EVP_DigestInit.pod

```diff
@@ -88,12 +90,25 @@ Cleans up digest context B<ctx> and frees up the space allocated to it.
 
 =item EVP_MD_CTX_ctrl()
 
+This is a deprecated option. EVP_MD_CTX_set_params() and EVP_MD_CTX_get_params()
```

Comment:
Do you mean this is a deprecated function? If so you don't actually mark it as deprecated with the deprecation macros.

Reply:
There were still a few references to this method in ssl3 code. I have converted them to use set_params and deprecated the function.